cosmic-core/test/integration/smoke/test_privategw_acl.py

```diff
@@ -307,7 +213,7 @@ def setup_infra(cls, redundant=False):
         cls.logger.debug("VPC '%s' created, CIDR: %s", cls.vpc2.name, cls.vpc2.cidr)
 
         cls.network2 = Network.create(cls.api_client,
-            cls.attributes['networks']['network2'],
+            cls.attributes['networks']['network3'],
```

Comment:
It's `['network3']` because we merged test cases in test_data.py, and we already have `['network2']` there
```
"networks": {
        "network1": {
            "name": "network1",
            "displaytext": "network1",
            "gateway": "10.1.1.1",
            "netmask": "255.255.255.0"
        },
        "network2": {
            "name": "network2",
            "displaytext": "network2",
            "gateway": "10.1.2.1",
            "netmask": "255.255.255.248"
        },
        "network3": {
            "name": "network3",
            "displaytext": "network3",
            "gateway": "10.2.1.1",
            "netmask": "255.255.255.0"
        },
        "private_gateways_network": {
            "name": "private_gateways_network",
            "displaytext": "private_gateways_network",
            "cidr": "172.16.1.0/24"
        }
    }
```